resolwe_bio/docker_images/rnaseq/assets/genialis.css

```diff
@@ -0,0 +1,135 @@
+body {
+  font-family: 'Roboto';
+  font-size: 12px;
+  font-style: normal;
+  color: #212121;
+  background-color: #fff;
+  margin: 2em 2% 4em 2%;
+  width: 80%!important;
```

Comment:
I would remove `width` and just update `margin` with it's final value `2em 18% 4em 2%`